main.go

```diff
@@ -305,6 +324,17 @@ func healthCheck(cfg config.AppConfig, logger log.Logger) error {
 	return nil
 }
 
+func printVersion(writer io.Writer) error {
+	var buffer bytes.Buffer
+	buffer.WriteString("Containerssh Version:")
+	buffer.WriteString(version)
+	buffer.WriteString("\n")
```

Comment:
```suggestion
	buffer.WriteString("v")
	buffer.WriteString(version)
	buffer.WriteString("\n")
```

Just a nitpick, I think a more compact output is best :)

Reply:
thanks @tsipinakis 
`It'd be better to read the info from the [buildinfo](https://pkg.go.dev/runtime/debug#ReadBuildInfo) API, perhaps of ContainerSSH and libcontainerssh version, which may not always be the same.`
As your opinion, the two versions may be different, but I specified the package name for now.
think it would be better to specify the version for each package in the future.